lockfile/src/python.rs

```diff
@@ -140,13 +146,9 @@ impl Parse for Poetry {
         path.file_name() == Some(OsStr::new("poetry.lock"))
     }
 
-    fn is_path_manifest(&self, path: &Path) -> bool {
-        path.file_name() == Some(OsStr::new("pyproject.toml"))
-    }
-
-    #[cfg(feature = "generator")]
-    fn generator(&self) -> Option<&'static dyn Generator> {
-        Some(&PoetryGenerator)
+    fn is_path_manifest(&self, _path: &Path) -> bool {
+        // `pyproject.toml` is handled by PyRequirements parser
+        false
```

Comment:
What if I explicitly specify the poetry ecosystem?

Shouldn't this by pyproject.toml and just use the same generator?

Reply:
Good catch. I didn't think of that case.